public/pages/DetectorsList/components/EmptyMessage/EmptyMessage.tsx

```diff
@@ -45,13 +51,27 @@ export const EmptyDetectorMessage = (props: EmptyDetectorProps) => (
           Reset filters
         </EuiButton>
       ) : (
-        <EuiButton
-          fill
-          href={`${PLUGIN_NAME}#${APP_PATH.CREATE_DETECTOR}`}
-          data-test-subj="add_detector"
-        >
-          Create detector
-        </EuiButton>
+        <EuiFlexGroup direction="row" gutterSize="m" justifyContent="center">
```

Comment:
Looks like the same code with `public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx`. Can we remove the duplicate code ?